backend/graphql/handler.ts

```diff
@@ -1,25 +1,68 @@
 import { executeQuery } from "../psql/queryExecute";
-import { QUERY_GET_ALL_DAOS } from "../psql/query";
-import { DaoType } from "./types";
-import { DaoItemType } from "../types";
-const { GraphQLObjectType, GraphQLList } = require("graphql");
+import { QUERY_GET_DAOS_COUNT, QUERY_PAGINATED_DAOS } from "../psql/query";
+import { DaosAndPageInfoType } from "./types";
+import { DaoItemType, DaoArgType, DaoAndPageResType } from "../types";
+const { GraphQLObjectType, GraphQLInt, GraphQLNonNull } = require("graphql");
+
+const validateRes = (totalDaosRes: any, paginateRes: any, args: DaoArgType) => {
```

Comment:
Could we use type on those arguments?

Reply:
It will be `any`. We don't know the type.